.travis.yml

```diff
@@ -6,4 +6,28 @@ rvm:
   - 2.0.0
   - 2.1.6
   - 2.2.2
-
+  - jruby-19mode
+  - jruby-9.0.0.0
+  - rbx-2
+jdk:
+  - openjdk7
```

Comment:
how about a matrix with jruby19+jdk7 and 9000+jdk8 ?

On Mon, Jul 27, 2015 at 2:22 PM, Jason Porter <notifications@github.com>
wrote:

> In .travis.yml
> <https://github.com/grosser/parallel/pull/141#discussion_r35588835>:
>
> > @@ -6,4 +6,28 @@ rvm:
> >    - 2.0.0
> >    - 2.1.6
> >    - 2.2.2
> > -
> > +  - jruby-19mode
> > +  - jruby-9.0.0.0
> > +  - rbx-2
> > +jdk:
> > +  - openjdk7
>
> There are some invoke dynamic changes in 7 and 8. I'd suggest at least
> testing those.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/grosser/parallel/pull/141/files#r35588835>.
>